src/sections/VotingSection.tsx

```diff
@@ -21,8 +21,13 @@ export const VotingSection = ({
   ballot,
   onVote,
 }: VotingSectionProps) => {
+  const [language, setLanguage] = useState<keyof TranslatableText>("en");
   const [captchaSaved, setCaptchaSaved] = useState(false);
-  const { t } = useTranslation();
+  const { t, i18n } = useTranslation();
+
+  useEffect(() => {
+    setLanguage(i18n.language.slice(0, 2) as never);
```

Comment:
Don't really like the `as never` but there probably isn't anything much better :/